### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by chunkybanana**

### CODE REVIEW

**Review by Lyxal — APPROVED** [review]

File: documents/knowledge/elements.yaml [hunk 1]
```diff
@@ -2452,7 +2452,7 @@
 - element: "⇩"
   name: Grade Down
   arity: 1
-  description: Indices of elements to sort in descending order / lowercase / decrement number twice
+  description: 'Indices of elements to sort in descending order / lowercase / decrement number twice'
```
Comment (ysthakur) [inline documents/knowledge/elements.yaml:2455]: Why the quotes? Was there an issue with the YAML?

File: documents/knowledge/elements.yaml [hunk 2]
```diff
@@ -3179,10 +3179,411 @@
...
+
+- element: "øṁ"
+  name: Vertical Mirror
+  description: Vertical Mirror - Split by newlines, mirror each line, join by newlines
+  arity: 1
+  overloads:
+    str: vertical_mirror(a)
+  vectorise: true
+  tests:
+    # TODO: MULTILINE TESTCASSES
```
Comment (ysthakur) [inline documents/knowledge/elements.yaml:3567]: YAML multiline strings can be done with `>-` if we want to turn the newlines into spaces or `|` if we want to keep the newlines.

**Review by ysthakur — APPROVED** [review]
Looks good, great work

File: documents/knowledge/elements.yaml (hunk 2 above)
Comment (chunkybanana) [reply documents/knowledge/elements.yaml:3567]: Ok, I'll do this in a bit.

File: documents/knowledge/elements.yaml (hunk 1 above)
Comment (chunkybanana) [reply documents/knowledge/elements.yaml:2455]: No clue.

**Pull request merged**